src/monitord/sign_log.c

```diff
@@ -13,7 +13,7 @@
 #include "os_crypto/sha1/sha1_op.h"
 #include "os_crypto/sha256/sha256_op.h"
 #include "monitord.h"
-#include <openssl/md5.h>
+#include <openssl/evp.h>
 #include <openssl/sha.h>
```

Comment:
Is this include still needed?

Reply:
It defines both `SHA_DIGEST_LENGTH` and `SHA256_DIGEST_LENGTH` so yes, it is still necessary to include it.